functions/data-mapping/src/data-mapping/routes.ts

```diff
@@ -0,0 +1,31 @@
+import { getDataMapping } from "./services";
+import express, { Request, Response } from "express";
+import { env } from "../env";
+
+const AI_MODEL = env.AI_MODEL;
+export class DataMappingRouter {
+  public router = express.Router();
+
+  constructor() {
+    this.registerRoutes();
+  }
+
+  private registerRoutes() {
+    this.router.post("/", async (req: Request, res: Response) => {
+      req.body.model = AI_MODEL;
```

Comment:
since only `getModels` function uses this value, can the service import this value instead of being passed through the request?